src/avram/migrator/alter_table_statement.cr

```diff
@@ -175,12 +175,25 @@ class Avram::Migrator::AlterTableStatement
     @fill_existing_with_statements << "ALTER TABLE #{@table_name} ALTER COLUMN #{column} SET NOT NULL;" unless nilable
   end
 
-  {% symbol_expected_message = "%s expected a symbol like ':user', instead got: '%s'" %}
+  macro symbol_expected_error(action, name)
+
```

Comment:
We may want to check if `name` is a `TypeDeclaration` here. It could be a `String` in some cases...

Reply:
OK, so I made the change (locally)  (but also accepted your smaller changes in Github. When I try and push my changes back up I get an error because "the tip of your branch is behind its remote counterpart. I think this might be because I accepted your changes. The error also says git pull to remedy the situation but is it a git pull from Avram or from my local github copy (as you can tell I'm a bit new to all of this :-)). cheers.

Reply:
You should be able to just run `git pull` and be good, or `git pull origin shortly-portly:improve-alter-migrations-error-msg` may be needed. But I'd say just try `git pull` first.

Reply:
Thanks Jeremy - I've just got it sorted and pushed a newer version that caters for Strings as you suggested. Hopefully we are good to go but if you have any further suggestions then just shout! cheers.